external/netutils/netlib_parsehttpurl.c

```diff
@@ -120,6 +120,7 @@ int netlib_parsehttpurl(const char *url, uint16_t *port, char *hostname, int hos
 			bytesleft--;
 		} else {
 			ret = -E2BIG;
+			return ret;
```

Comment:
How about 'return -E2BIG;'
Also, there is another error case with -ENOENT.

Reply:
done